src/report_generator.cpp

```diff
@@ -23,28 +25,11 @@ namespace
 // Generic utilities that are useful and do not rely on context or types from our domain (issue-list processing)
 // =============================================================================================================
 
-auto format_time(std::string const & format, std::tm const & t) -> std::string {
-   std::string s;
-   std::size_t maxsize{format.size() + 256};
-  //for (std::size_t maxsize = format.size() + 64; s.size() == 0 ; maxsize += 64)
-  //{
-      std::unique_ptr<char[]> buf{new char[maxsize]};
-      std::size_t size{std::strftime( buf.get(), maxsize, format.c_str(), &t ) };
-      if(size > 0) {
-         s += buf.get();
-      }
- // }
-   return s;
-}
-
-auto utc_timestamp() -> std::tm const & {
-   static std::time_t t{ std::time(nullptr) };
-   static std::tm utc = *std::gmtime(&t);
-   return utc;
-}
+static auto const timestamp{std::chrono::floor<std::chrono::seconds>(std::chrono::system_clock::now())};
 
 // global data - would like to do something about that.
-static std::string const build_timestamp{format_time("Revised %Y-%m-%d at %H:%M:%S UTC\n", utc_timestamp())};
+static std::string const build_date{std::format("{:%F}", timestamp)};
+static std::string const build_timestamp{std::format("Revised {} at {:%T} UTC\n", build_date, timestamp)};
 
 static std::string const maintainer_email{"lwgchair@gmail.com"};
```

Comment:
Pre-existing: `static` is redundant for `const` objects. Doubly so for `const` objects defined in an unnamed namespace.

Reply:
Yeah, I was going to fix that on `master` anyway, not just as part of this c++20 branch. We don't need triple redundancy here.

Reply:
Done with bb7b2d57d366eae00dc7747898f0d233fb29a62c